Maria-Edgeworth-Letters-Project/me-tei
# Validation errors

## What does this Pull Request do?

This PR standardizes spacing across all records. More importantly, it also addresses validation errors in four existing XML files:

- Edgeworth_NLS_23130_1814_11_10.xml
- Edgeworth_Morgan_301751_1816_04_03.xml
- Edgeworth_Geneve_33_2_unknown_04.xml
- Edgeworth_Beinecke_330_1836.xml

## How should this be tested?

Briefly review a few records with the spacing changes to ensure that these changes are appropriate. Look closely at the four records mentioned above. Ask yourself:
- Do the changes alter the meaning of the encoding? Or are they inaccurately representing the digitized letter?
- Does this follow the project's encoding guidelines - https://docs.google.com/document/d/1mrmfs6BHDj7DB9iRCd6B_hw03n2srZ4y/edit

## Additional Notes

None.
